contracts/PrisonersDilemma.sol

```diff
@@ -68,9 +68,15 @@ contract PrisonersDilemma {
         players[msg.sender].choice = choice;
 
         emit PlayerSelectedChoice(msg.sender);
+
+        //TOOD: I need to add logic here that compares the selected choice of this user and the other user
```

Comment:
Seems I need to know what players are already apart of the contract. Since 'mapping' doesn't show the keys I need to track them somewhere. 

I could do a few things:

1. abstract logic away and create my own Map that uses an array and a mapping to look up Players
2. use an array to directly store the players apart of the contract in
3. resort to player1 player2. The problem with this is that there is no label of 1 v 2

Reply:
Hi Kaylin and Cody,

Thanks again for sending those resources.  
My Dad and I took a look at the proposal along with the other two projects you sent to us. It's fair to say we are both impressed; you both know how to utilize space to it's fullest potential. 

We assuredly want to be conscious of other's time and think it best if we continue to learn and research as to developer a clearer picture of the cost and scope.

We want to thank you for the time taken to speak with us about the overall process, as well as the work you shared.

Reply:
https://ethereum.stackexchange.com/questions/15337/can-we-get-all-elements-stored-in-a-mapping-in-the-contract

Reply:
https://github.com/ejwessel/PrisonersDilemma/pull/7